osh/worker/tasks/task_build.py

```diff
@@ -93,19 +109,13 @@ def notification(cls, hub, conf, task_info):
         hub.worker.email_task_notification(task_info["id"])
 
 
-class DiffBuild(Build, TaskBase):
-    def __init__(self, *args, **kwargs):
-        Build.__init__(self)
-        TaskBase.__init__(self, *args, **kwargs)
+class DiffBuild(Build):
+    enabled = True
 
 
 class MockBuild(Build, TaskBase):
```

Comment:
Now we can remove `, TaskBase` also here.

Reply:
Thank you! I forgot to change that line as well.  Addressed in 9ea6b52/